### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by michelleN**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/documentation.md [hunk 1]
```diff
@@ -7,9 +8,5 @@ assignees: ''
 ---
 
 **Please describe the what should be documented
```
Comment (ksubrmnn) [inline .github/ISSUE_TEMPLATE/documentation.md:10]: Remove "the"

File: .github/ISSUE_TEMPLATE/documentation.md [hunk 2]
```diff
@@ -7,9 +8,5 @@ assignees: ''
...
-<!-- A clear and concise description of what scenario should be documented and why. -->
 
-**Suggestion for where the documentation should live. Should it be part of an existing 
-file or directory? Should there be a new file? Should it be part of the demo? Should it be
-part of the help text for the CLI?**
-<!-- A path or link here -->
+**Please suggest where in the repo should be documented
```
Comment (ksubrmnn) [inline .github/ISSUE_TEMPLATE/documentation.md:12]: Suggestion:
Please suggest where in the repo the document should be located

File: .github/ISSUE_TEMPLATE/documentation.md (hunk 1 above)
Comment (michelleN) [reply .github/ISSUE_TEMPLATE/documentation.md]: thank you

File: .github/ISSUE_TEMPLATE/documentation.md (hunk 2 above)
Comment (michelleN) [reply .github/ISSUE_TEMPLATE/documentation.md]: thank you

**Pull request merged**